apps/wallet-mobile/src/components/Button/Button.tsx

```diff
@@ -154,6 +155,9 @@ const useStyles = () => {
     buttonDisabled: {
       opacity: 0.5,
     },
+    buttonDisabledText: {
+      color: color.gray_cmin,
```

Comment:
text, should be `text_..min`

Reply:
confirm if is the case for the buttons as well.

Reply:
text_gray_min is not created 👀

will have a look

Reply:
done